arkouda/dataframe.py

```diff
@@ -21,7 +19,7 @@
 from arkouda.pdarraysetops import concatenate, intersect1d, in1d
 
 from arkouda.pdarrayIO import save_all, load_all
-from arkouda.dtypes import int64 as akint64, uint64 as akuint64, float64 as akfloat64, bool as akbool
+from arkouda.dtypes import int64 as akint64, float64 as akfloat64, bool as akbool
```

Comment:
i noticed you removed uint64 here... do we need it?

Reply:
@mhmerrill - it was unused, so I removed it